Command/GenerateDoctrineCrudCommand.php

```diff
@@ -261,6 +263,37 @@ protected function updateRouting(QuestionHelper $questionHelper, InputInterface
                 '',
             );
         }
+
+        // second, import the bundle's routing.yml file from the application's routing.yml file
+        $routing = new RoutingManipulator($this->getContainer()->getParameter('kernel.root_dir').'/config/routing.yml');
+        try {
+            $ret = $auto ? $routing->addResource($bundle->getName(), $format) : false;
```

Comment:
the `$format` here should always be `yml`, correct? Because we want to import `@YourBundle/Resources/config/routing.yml`?